cmake/python.cmake

```diff
@@ -121,11 +130,18 @@ message(STATUS "PYTHON_MODULE_EXTENSION: ${PYTHON_MODULE_EXTENSION}")
 message(STATUS "PYTHON_IS_DEBUG: ${PYTHON_IS_DEBUG}")
 message(STATUS "PYTHON_SIZEOF_VOID_P: ${PYTHON_SIZEOF_VOID_P}")
 message(STATUS "PYTHON_LIBRARY_SUFFIX: ${PYTHON_LIBRARY_SUFFIX}")
-message(STATUS "PYTHON_LIBDIR: ${PYTHON_LIBDIR}")
 message(STATUS "PYTHON_MULTIARCH: ${PYTHON_MULTIARCH}")
 
+# Python, especially on the mac, doesn't always return the right PYTHON_LIBDIR,
+# so in the case that libdir is not able to be found, use a relative path from the include_dir
+IF(NOT EXISTS ${PYTHON_LIBDIR})
+    get_filename_component(_PYTHON_LIBDIR ${PYTHON_INCLUDE_DIR} PATH)
+    get_filename_component(_PYTHON_LIBDIR ${_PYTHON_LIBDIR} PATH)
+    set(PYTHON_LIBDIR "${_PYTHON_LIBDIR}/lib")
+endif()
+message(STATUS "PYTHON_LIBDIR: ${PYTHON_LIBDIR}")
 
-# Make sure the Python has the same pointer-size as the chosen compiler
+    # Make sure the Python has the same pointer-size as the chosen compiler
```

Comment:
Did you accidentally change the spacing on this line?

Reply:
Yes, sorry, removed it in the latest update that I pushed just now